TypoSearch.py

```diff
@@ -115,29 +112,30 @@ def main():
                         required=True)
     parser.add_argument("-table", action="store", dest="table", help="Name of table to search in", required=True)
     parser.add_argument("-rank", action="store", dest="rank", help="Rank level to start searching at", required=True)
+    parser.add_argument("-typo", action = "store", dest = "typo",
```

Comment:
can we call this argument typo_limit or something else more descriptive?

Reply:
This argument has been changed to typo_limit.